server/utils/nunjucksSetup.ts

```diff
@@ -79,5 +81,109 @@ export default function nunjucksSetup(app: express.Application): nunjucks.Enviro
       checked: item === value,
     }))
   })
+
+  njkEnv.addFilter('toCourtTypeFilterCheckboxes', (courtTypes: CourtType[], allCourtsFilter: AllCourtsFilter) => {
+    const courtTypeItems = courtTypes.map((courtType: CourtType) => {
+      return {
+        value: courtType.courtType,
+        text: courtType.courtName,
+        checked: allCourtsFilter.courtTypeIds.includes(courtType.courtType),
+      }
+    })
+    return {
+      idPrefix: 'courtType',
+      name: 'courtType',
+      classes: 'govuk-checkboxes--small',
+      fieldset: {
+        legend: {
+          text: 'Court Types',
+          classes: 'govuk-fieldset__legend--m',
+        },
+      },
+      items: courtTypeItems,
+    }
+  })
+
+  njkEnv.addFilter('toActiveFilterRadioButtons', (allCourtsFilter: AllCourtsFilter) => {
+    return {
+      idPrefix: 'active',
+      name: 'active',
+      classes: 'govuk-radios--inline',
+      fieldset: {
+        legend: {
+          text: 'Active?',
+          classes: 'govuk-fieldset__legend--l',
+        },
+      },
+      hint: {
+        text: 'Show only open or closed courts',
+      },
+      items: [
+        {
+          value: '',
+          text: 'All',
+          checked: allCourtsFilter.active === null,
+        },
+        {
+          value: true,
+          text: 'Open',
+          checked: allCourtsFilter.active === true,
+        },
+        {
+          value: false,
+          text: 'Closed',
+          checked: allCourtsFilter.active === false,
+        },
+      ],
+    }
+  })
+
+  njkEnv.addFilter('toCourtListFilter', (courtTypes: CourtType[], allCourtsFilter: AllCourtsFilter) => {
```

Comment:
Generates GDS style filter header for AllCourtsFilter based on the MOJ filter spec. here: https://moj-design-system.herokuapp.com/patterns/filter-a-list